fixes #64: javavscpp explanation added (#69)
comp129/comp55: docs/labs/JavavsCPP.md

but you didn't create space for the Fraction object itself.
By calling ```new```.

The reason why you hear people saying Java doesn't have pointers,
is because of semantics.

> The term "pointer" is strongly associated with C/C++ concepts,

Reviewer: ```suggestion
The term ***pointer*** is strongly associated with C/C++ concepts,
```

which are variables that explicitly store memory addresses,
and which can be modified arithmetically to point to any arbitrary address.
But in Java, pointers exist as an **implementation detail for references**,

> In other words, Java ***do*** have pointers,

Reviewer: ```suggestion
In other words, Java ***does*** have pointers,
```

> but they can't do pointer arithmetics like C/C++ can.

Reviewer: ```suggestion
but they can't be manipulated via pointer arithmetic like C/C++ can.
```

> If you want to read more, check on [this wiki page](https://en.wikipedia.org/wiki/Pointer_%28computer_programming%29#Making_pointers_safer)

Reviewer: ```suggestion
If you want to read more, visit [this wiki page](https://en.wikipedia.org/wiki/Pointer_%28computer_programming%29#Making_pointers_safer)
```

> to understand a bit more about this.

Reviewer: ```suggestion
to understand a bit more about pointers.
```
